utils/cosmos/iscn/iscn.type.ts

```diff
@@ -24,6 +24,9 @@ export interface ISCNRegisterPayload {
   stakeholders?: any[];
   recordNotes?: string;
   memo?: string;
+  inLanguage?: string;
+  thumbnailUrl?: string;
+  isbn?: string | number;
```

Comment:
Can we just cast all as either string or number? having both is not good for parsing